airflow/providers/common/sql/hooks/sql.py

```diff
@@ -147,14 +147,23 @@ class DbApiHook(BaseHook):
     default_conn_name = "default_conn_id"
     # Override if this db supports autocommit.
     supports_autocommit = False
+    # Override if this db supports executemany.
+    supports_executemany = False
     # Override with the object that exposes the connect method
     connector: ConnectorProtocol | None = None
     # Override with db-specific query to check connection
     _test_connection_sql = "select 1"
     # Default SQL placeholder
     _placeholder: str = "%s"
 
-    def __init__(self, *args, schema: str | None = None, log_sql: bool = True, **kwargs):
+    def __init__(
+        self,
+        *args,
+        schema: str | None = None,
+        log_sql: bool = True,
+        fast_executemany: bool = False,
```

Comment:
Nothing seems to use this argument anywhere?

Reply:
I removed it